src/components/Modals/BuyBuzzModal.tsx

```diff
@@ -1,149 +1,223 @@
+                  setCustomAmount(undefined);
+                  setError('');
+                  setSelectedPrice(selectedPackage ?? null);
+                }}
+              >
+                {packages.map((buzzPackage, index) => {
+                  const price = (buzzPackage.unitAmount ?? 0) / 100;
+
+                  return (
+                    <Chip
+                      key={buzzPackage.id}
+                      value={buzzPackage.id}
+                      classNames={{ label: classes.chipLabel, iconWrapper: classes.chipCheckmark }}
+                      variant="filled"
+                    >
+                      <Group align="center">
+                        <Text color="accent.5">
+                          <IconBolt
+                            color="currentColor"
+                            fill="currentColor"
+                            style={{ verticalAlign: 'middle' }}
+                          />
                         </Text>
-                      </Center>
-                    </Card.Section>
-                  </Card>
-                );
-              })}
-        </SimpleGrid>
+                        <Stack spacing={0}>
+                          {price ? (
+                            <>
+                              <Text size="lg" color="white" weight={590}>
+                                {buzzPackage.name ?? `Tier ${index + 1}`}
+                                <Text sx={{ fontVariantNumeric: 'tabular-nums' }} span>
+                                  {` ($${price.toFixed(2)})`}
+                                </Text>
+                              </Text>
+                              <Text size="md">
+                                {buzzPackage.buzzAmount
+                                  ? buzzPackage.buzzAmount.toLocaleString()
+                                  : 0}{' '}
+                                Buzz
+                              </Text>
+                            </>
+                          ) : (
+                            <>
+                              <Text size="lg" color="white" weight={590}>
+                                Custom amount
+                              </Text>
+                              <Text size="md" color="dimmed">
+                                You choose how much Buzz you want to buy
+                              </Text>
+                            </>
+                          )}
+                        </Stack>
+                      </Group>
+                    </Chip>
+                  );
+                })}
+              </Chip.Group>
+
+              {selectedPrice && !selectedPrice.unitAmount && (
+                <NumberInput
+                  placeholder="Minimum $5 USD"
+                  variant="filled"
+                  icon={<CurrencyIcon currency="USD" size={18} fill="transparent" />}
+                  value={customAmount}
+                  min={5}
```

Comment:
`499` here. also, might be good to keep a const with the min. custom purchase amount in case it changes down the line